console/src/main/java/org/georchestra/console/ws/backoffice/roles/RolesController.java

```diff
@@ -519,12 +523,21 @@ private Role createRoleFromRequestBody(ServletInputStream is) throws IOException
 				throw new IllegalArgumentException(RoleSchema.COMMON_NAME_KEY + " is required" );
 			}
 
+			// Capitalize role name and check format
+			commonName = commonName.toUpperCase();
+			Pattern p = Pattern.compile("[A-Z0-9_]+");
+			if(!p.matcher(commonName).matches())
+				throw new IllegalArgumentException(RoleSchema.COMMON_NAME_KEY +
+						" must only contains upper case letter, digits and underscore" );
```

Comment:
"should only contain uppercased letters, digits and underscores (not leading nor ending with)"